.travis/deploy.sh

```diff
@@ -5,35 +5,7 @@ set -x
 
 for image in app editor web gradle analysis;
 do
-  docker push "quay.io/azavea/driver-${image}:${TRAVIS_COMMIT:0:7}"
-  docker tag "quay.io/azavea/driver-${image}:${TRAVIS_COMMIT:0:7}" "quay.io/azavea/driver-${image}:latest"
-  docker push "quay.io/azavea/driver-${image}:latest"
+  docker push "quay.io/azavea/driver-${image}:${TRAVIS_TAG}"
```

Comment:
I think that if we remove `on: master` above, this will work. Otherwise, I think the `TRAVIS_TAG` environment variable will be unset when a build is triggered due to changes to `master`.

Reply:
See above; my understanding from the docs is that `tags: true` will prevent this from running if `TRAVIS_TAG` is unset.

Reply:
Yeah, this seems to back that up too:

> tags: true: deployment is triggered if and only if $TRAVIS_TAG is set. Depending on your workflow, you may set $TRAVIS_TAG explicitly, even if this is a non-tag build when it was initiated. This causes the branch condition to be ignored.